app/lib/shared/handler_helpers.dart

```diff
@@ -230,14 +230,18 @@ shelf.Handler _userSessionWrapper(shelf.Handler handler) {
     if (isPrimaryHost &&
         isAllowedForSession &&
         request.headers.containsKey(HttpHeaders.cookieHeader)) {
-      final sessionId = session_cookie.parseSessionCookie(
-        request.headers[HttpHeaders.cookieHeader],
-      );
-      if (sessionId != null && sessionId.isNotEmpty) {
-        final sessionData = await accountBackend.lookupSession(sessionId);
-        if (sessionData != null) {
-          registerUserSessionData(sessionData);
+      try {
+        final sessionId = session_cookie.parseSessionCookie(
+          request.headers[HttpHeaders.cookieHeader],
+        );
+        if (sessionId != null && sessionId.isNotEmpty) {
+          final sessionData = await accountBackend.lookupSession(sessionId);
+          if (sessionData != null) {
+            registerUserSessionData(sessionData);
+          }
         }
+      } catch (e, st) {
+        logger.warning('Unable to process session cookie.', e, st);
```

Comment:
I think this needs to be `logger.severe` or something more than `warning`.

Afaik, warning will not trigger reporting to stack driver error reporting, only `severe` and higher, see:
https://pub.dev/documentation/appengine/latest/appengine/useLoggingPackageAdaptor.html